retrieverdash/dashboard_script/dashboard_script.py

```diff
@@ -0,0 +1,58 @@
+import json
+import os
+from multiprocessing import Pool
+
+from filelock import FileLock
+from retriever import datasets
+
+from status_dashboard_tools import (
```

Comment:
I really do not feel comfortable with this .🙂 let's change it to one line.

Reply:
Hehe. Okay :smile: